tests/Tests/200__monitor_and_manage/200__metrics/200__metrics.robot

```diff
@@ -1,26 +1,133 @@
 *** Settings ***
 Resource    ../../../Resources/ODS.robot
+Test Setup  Begin Matrics Web Test
+
 
 *** Variables ***
+@{RECORD_GROUPS}    Availability Metrics    SLOs - JupyterHub    SLOs - ODH Dashboard    SLOs - RHODS Operator    SLOs - Traefik Proxy    Usage Metrics
 
-@{RECORD_GROUPS}  Availability Metrics  SLOs - JupyterHub  SLOs - ODH Dashboard  SLOs - RHODS Operator  SLOs - Traefik Proxy  Usage Metrics
+@{ALERT_GROUPS}     Builds                  DeadManSnitch    RHODS-PVC-Usage    SLOs-haproxy_backend_http_responses_total    SLOs-probe_success
 
-@{ALERT_GROUPS}  Builds  DeadManSnitch  RHODS-PVC-Usage  SLOs-haproxy_backend_http_responses_total  SLOs-probe_success
 
 *** Test Cases ***
-
 Test Existence of Prometheus Alerting Rules
-  [Tags]  Smoke  Sanity  ODS-509
-  Check Prometheus Alerting Rules
+    [Tags]    Smoke
+    ...       Sanity
+    ...       ODS-509
+    Check Prometheus Alerting Rules
 
 Test Existence of Prometheus Recording Rules
-  [Tags]  Smoke  Sanity  ODS-510
-  Check Prometheus Recording Rules
+    [Tags]    Smoke
+    ...       Sanity
+    ...       ODS-510
+    Check Prometheus Recording Rules
+
+Test Metric "Notebook CPU Usage" On ODS Prometheus
+    [Documentation]    Verifing the notebook cpu usage showing on RHODS promethues
+    [Tags]    Sanity
+    ...       Tier1
+    ...       ODS-178
+    ${cpu_usage_before} =    Read Current CPU Usage
+    Run Jupyter Notebook For 5 Minutes
+    ${cpu_usage_after} =    Read Current CPU Usage
+    Should Not Be Equal    ${cpu_usage_before}    ${cpu_usage_after}
+    Close All Browsers
```

Comment:
I think `Close All Browsers` should be in the suite/test teardown

Reply:
actually, our all test cases do not use web browser just this one we are using that is why I have used it here not in teardown

Reply:
I think that line won't be executed if the cpu readings are equal. It should go to the teardown. It's fine to have it in the Suite Teardown, even if just that test is using a browser